image/cli/mascli/functions/gitops_cluster

```diff
@@ -467,6 +475,11 @@ function gitops_cluster() {
   echo_reset_dim "Install the CrowdStrike Falcon Operator ........... ${COLOR_MAGENTA}${INSTALL_FALCON_OPERATOR}"
   reset_colors
 
+  echo "${TEXT_DIM}"
+  echo_h2 "Cluster Logging Operator" "    "
+  echo_reset_dim "Install the Cluster Logging Operator ........... ${COLOR_MAGENTA}${CLUSTER_LOGGING_OPERATOR_INSTALL}"
```

Comment:
Should we also echo out cluster_logging_operator_use_syslog_forwarder?

Reply:
@boonware As we are setting it in the j2 template I think it would be useful to know what the value is

Reply:
@whitfiea Change made as recommended.